src/components/Header/Nav/NavBar.tsx

```diff
@@ -1,6 +1,11 @@
 import React from "react"
 import { ProductsNav, SubProductsNav } from "./config"
-import { NavBarView } from "./NavBarView"
+import styles from "./navBar.module.css"
```

Comment:
NavBar and NavBarView were separate for visual testing purposes. Since we are not doing that in here, I merged them into one file to keep it simple